### ISSUE TITLE
Cannot import template 6.0 into Zabbix 6.0.30

### ISSUE THREAD

**Issue by ioctl2** [albin-lindstrom/zabbix-edgemax-template#20]
I'm trying to import the 6.0 template into 6.0 LTS, and it's throwing this error: `    Cannot import template "EdgeMAX SNMPv2", linked template "Generic by SNMP" does not exist.` This is a fresh deployment of Zabbix, at version 6.0.30.

I could be doing something wrong, but I did not have an issue with the 6.4 template on 7.0LTS, so I am not sure.

**Comment by kodiakastu** [comment]
Generic by SNMP was provided by the earlier versions on fresh install. Are you able to check if there is such template or a similar named template?

**Comment by ioctl2** [comment]
I've messed with importing various templates since posting, so my instance is no longer representative of a clean install. But I do not recall `Generic by SNMP` being available on a clean install. I think that instead it makes available `'Module Generic SNMP'`. I will do a clean install in a test VM for a sanity check.

**Comment by ioctl2** [comment]
I just did a clean install and `Generic by SNMP` is indeed available, and the template imported fine. Not sure what happened with the other install, but it looks like the issue was installation-specific.

**Comment by kodiakastu** [comment]
Worst case, export your 6.0.30 ``Generic by SNMP`` to the good machine. Not sure either why it was missing.

**Comment by ioctl2** [comment]
Yeah, it's a bit odd, but thankfully the fix is simple. Apologies for the unnecessary issue.
